app/main.go

```diff
@@ -92,6 +92,9 @@ var (
 	boltdbPath = kingpin.Flag("boltdb-path", "path to BoltDB store, e.g. /data/bolt.db").Envar("BOLTDB_PATH").String()
 
 	ver = kingpin.Flag("version", "print the current version revision").Bool()
+
+	monitoringBackends = kingpin.Flag(
```

Comment:
If I specify `prometheus` as a stats collector, the Stackdriver collector is still getting initialized, is this by design?

Reply:
If not - I think it's a good idea to set `stackdriver` as default value here (to avoid breaking changes) and fail fast if the arguments supplied are not enough to run.

Reply:
Stackdriver is now the default.